doc/dune

```diff
@@ -14,8 +14,6 @@
   ;   + config/coq_config.py
   ;   + tools/coqdoc/coqdoc.css
   (package rocq-runtime)
-  (package coq-core)
-  (package rocq-core)
```

Comment:
@Zimmi48 your fix may just be a matter of readding those tow lines (and two other instances below)

Reply:
Yep, I'm testing this right now.

Reply:
Done in #20826.